podcast.rss

```diff
@@ -7341,5 +7341,349 @@
+      <itunes:duration>10:43</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>1-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>627.സൂറഃ നബ പാരായണം-2_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #627 💠Surah An-Naba 💠 *പാരായണം* Part -2 (ആയത്ത്&zwnj; 20 മുതൽ 40 വരെ) بِسْمِ اللَّـهِ الرَّحْمَـٰنِ الرَّحِيمِ إِنَّ جَهَنَّمَ كَانَتْ مِرْصَادًا ﴿٢١﴾ لِّلطَّاغِينَ مَآبًا ﴿٢٢﴾ لَّابِثِينَ فِيهَا أَحْقَابًا ﴿٢٣﴾ لَّا يَذُوقُونَ فِيهَا بَرْدًا وَلَا شَرَابًا ﴿٢٤﴾ إِلَّا حَمِيمًا وَغَسَّاقًا ﴿٢٥﴾ جَزَاءً وِفَاقًا ﴿٢٦﴾ إِنَّهُمْ كَانُوا لَا يَرْجُونَ حِسَابًا ﴿٢٧﴾ وَكَذَّبُوا بِآيَاتِنَا كِذَّابًا ﴿٢٨﴾ وَكُلَّ شَيْءٍ أَحْصَيْنَاهُ كِتَابًا ﴿٢٩﴾ فَذُوقُوا فَلَن نَّزِيدَكُمْ إِلَّا عَذَابًا ﴿٣٠﴾إِنَّ لِلْمُتَّقِينَ مَفَازًا ﴿٣١﴾ حَدَائِقَ وَأَعْنَابًا ﴿٣٢﴾ وَكَوَاعِبَ أَتْرَابًا ﴿٣٣﴾ وَكَأْسًا دِهَاقًا ﴿٣٤﴾ لَّا يَسْمَعُونَ فِيهَا لَغْوًا وَلَا كِذَّابًا ﴿٣٥﴾ جَزَاءً مِّن رَّبِّكَ عَطَاءً حِسَابًا ﴿٣٦﴾ رَّبِّ السَّمَاوَاتِ وَالْأَرْضِ وَمَا بَيْنَهُمَا الرَّحْمَـٰنِ ۖ لَا يَمْلِكُونَ مِنْهُ خِطَابًا ﴿٣٧﴾ يَوْمَ يَقُومُ الرُّوحُ وَالْمَلَائِكَةُ صَفًّا ۖ لَّا يَتَكَلَّمُونَ إِلَّا مَنْ أَذِنَ لَهُ الرَّحْمَـٰنُ وَقَالَ صَوَابًا ﴿٣٨﴾ ذَٰلِكَ الْيَوْمُ الْحَقُّ ۖ فَمَن شَاءَ اتَّخَذَ إِلَىٰ رَبِّهِ مَآبًا ﴿٣٩﴾ إِنَّا أَنذَرْنَاكُمْ عَذَابًا قَرِيبًا يَوْمَ يَنظُرُ الْمَرْءُ مَا قَدَّمَتْ يَدَاهُ وَيَقُولُ الْكَافِرُ يَا لَيْتَنِي كُنتُ تُرَابًا ﴿٤٠﴾ 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/2-vqd-podcast-unit78.mp3" type="audio/mp3" length="2331665"/>
+      <itunes:duration>11:48</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>2-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>628.സൂറഃ നബ ആമുഖം _a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #628💠Surah An-Naba 💠 *ആമുഖം* 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/3-vqd-podcast-unit78.mp3" type="audio/mp3" length="2651490"/>
+      <itunes:duration>12:07</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>3-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>629.സൂറഃ നബ ആയത്ത് 1-5 Part 1_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #629 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; 1 -5* വിവരണം -1 عَمَّ يَتَسَآءَلُونَ ഏതിനെപ്പറ്റിയാണ് അവർ പരസ്പരം ചോദിച്ചുകൊണ്ടിരിക്കുന്നത്? عَنِ ٱلنَّبَإِ ٱلْعَظِيمِ അതേ ആ ഭയങ്കരമായ വാര്&zwj;ത്തയെപ്പറ്റി. ٱلَّذِى هُمْ فِيهِ مُخْتَلِفُونَ അവർ ഭിന്നാഭിപ്രായക്കാരായ ഒന്നിനെ പറ്റി. كَلَّا سَيَعْلَمُونَ അങ്ങനെയല്ല്ല അവർ വഴിയെ അറിയും. ثُمَّ كَلَّا سَيَعْلَمُونَ വീണ്ടും അങ്ങനെയല്ല്ല അവർ വഴിയെ അറിയും. 🔸Admin Team Velicham Onlive🔸</p>]]>
```

Comment:
@febincp6 comparing with previous chapters, the description seems missing a few tags which would be essential to keep the formatting.

Reply:
resolved